Assets/Scripts/VFX/RenderSettings/TimeLightingSettings.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[System.Serializable]
+public struct TimeLightingSettings 
+{
```

Comment:
what is the difference between timelightingsettings and timelightingsettingsdata?